src/components/UpcomingEventsList.tsx

```diff
@@ -56,10 +59,11 @@ const UpcomingEventsList = () => {
     const formattedEvents = orderedEvents.map((event) => {
       const dateRange =
         event.startDate === event.endDate
-          ? dateParse(event.startDate).toLocaleDateString()
-          : `${dateParse(event.startDate).toLocaleDateString()} - ${dateParse(
-              event.endDate
-            ).toLocaleDateString()}`
+          ? formatDate(dateParse(event.startDate), locale)
+          : `${formatDate(dateParse(event.startDate), locale)} - ${formatDate(
+              dateParse(event.endDate),
+              locale
+            )}`
```

Comment:
Would update this to use `getLocaleTimestamp` from `time.ts`... it accepts the `locale`, (of `Lang` type) then `timestamp` as a string (then optional `options` override argument which may be able to be left out here)

ie. `getLocaleTimestamp(locale! as Lang, dateParse(event.startDate).getTime())`

`locale!` force unwraps it, since we know we'll get a locale back from `useRouter`... and `as Lang` casts it to the expected type as defined inside `getLocaleTimestamp`.

^ With this, we need to fix the `locale` instantiation (see above), and also import `Lang` type above:

`import type { CommunityConference, Lang } from "@/lib/types"`